npm-shrinkwrap.json

```diff
@@ -104,6 +104,11 @@
         }
       }
     },
+    "@balena/dockerignore": {
```

Comment:
`@balena/dockerignore` does not have any additional dependencies, so it is not a big deal to temporarily have it in addition to `zeit/dockerignore` (until the next major version release).